### ISSUE TITLE
Change wording when deleting selected history items vs. all history items

### ISSUE THREAD

**Issue by Frans-Lukas** [kiwix/kiwix-android#2098]
**Is your feature request related to a problem? Please describe.**

When deleting history items from the history view, the confirmation dialog always says "**Delete history?**". I argue that this is not specific enough because there is a difference of deleting all items, or just the selected items which should be made more clear in the confirmation dialog.

**Describe the solution you'd like**

When trying to delete all items it should ask "**Delete all history?**"/"**Clear all history?**" and when trying to delete the selected history items it should ask "**Delete selected history?**"/"**Delete 10 selected history items?**" 

**Describe alternatives you've considered**

Keep it as is. I think some distinction should be made between the two dialogs.

**Additional context**

Current dialog text:

**Deleting all items**
![Screenshot_20200519-095949](https://user-images.githubusercontent.com/22193232/82300846-b13f4500-99b7-11ea-9bf0-4fa4475be262.jpg)

**Deleting selected items**
![Screenshot_20200519-095957](https://user-images.githubusercontent.com/22193232/82300864-b69c8f80-99b7-11ea-9482-45948de22621.jpg)

**Comment by macgills** [comment]
> "Delete 10 selected history items?"

That should be ruled out because we want to avoid pluralised strings eg "Delete 1 **item**" "Delete 2 **items**" as supporting this across all languages is a bit complicated.

>"Delete all history?"/"Clear all history?"

This seems ambiguous to me because we delete the "currently viewed" history, ie all history if "showAllHistory" is toggled or "current book history" if it is not and to communicate that to a user I feel gets too wordy and more likely to confuse.

> "Delete selected history?"

Is the only one that has my support right now but I could hear arguments

**Comment by Frans-Lukas** [comment]
I didn't know it only deleted the items you are currently viewing. I just assumed it deleted everything.  Which I think is telling about the clarity of the dialog text. 

What about "**Delete history from Europa Universalis 4 Wiki?**" or "**Europe Universalis 4 Wiki: Delete history from this book?**"?

and "**Delete history from all books?**" when all history is viewed.

**Comment by macgills** [comment]
>only deleted the items you are currently viewing.

I am recalling this from memory, may be prudent to look at the code.

All the following seem acceptable to me

>"Delete history from Europa Universalis 4 Wiki?"

>"Delete history from all books?"

>"Delete selected history?"
